SPECS-EXTENDED/perl-Test-Simple/perl-Test-Simple.spec

```
@@ -0,0 +1,776 @@
+%if ! (0%{?rhel})
+%bcond_without perl_Test_Simple_enables_optional_test
+%else
+%bcond_with perl_Test_Simple_enables_optional_test
+%endif
```

Same request regarding distro macros.
```suggestion
%bcond_without perl_Test_Simple_enables_optional_test
```

We might as well completely remove the `bcond` and just assume `perl_Test_Simple_enables_optional_test` is always true.